crux-bench/src/crux/bench.clj

```diff
@@ -44,6 +45,29 @@
 (defmacro ^{:style/indent 1} run-bench [bench-type & body]
   `(run-bench* ~bench-type (fn [] ~@body)))
 
+(defn post-to-slack [message]
+  (when (System/getenv "SLACK_URL")
+    (client/post (-> (System/getenv "SLACK_URL")
+                      (clojure.data.json/read-str)
+                      (get "slack-url"))
+                 {:body (json/write-str {:text message})
+                  :content-type :json})))
+
+(defn format-and-post-results-to-slack [result]
```

Comment:
whenever you've got `and` in a function name, that's usually a clue that the function's doing more than one thing :wink: